examples/mlflow_artifacts/docker-compose.yml

```diff
@@ -1,6 +1,9 @@
 version: "3"
 services:
   minio:
+    # Releases for 10/30/23 and 11/1/23 are broken.
```

Comment:
leaving this as an indicator of what to change it back to in case it's not one of us reverting this pin.